### ISSUE TITLE
Potions won't be pulled by logistical pipes from chests

### ISSUE THREAD

**Issue by Sly0511** [mekanism/Mekanism#7472]
### Issue description

I am trying to pull from chests potions to automate brewing stands but it just won't do it

### Steps to reproduce

![image](https://user-images.githubusercontent.com/47339373/167295013-5f30e9ba-b180-48a9-92f4-c41d0f4eff73.png)

### Minecraft version

1.16.x or earlier (No longer being developed)

### Forge version

36.2.34

### Mekanism version

Older

**Comment by leadwolf32** [comment]
Have you tested other input methods, say using the RS exporter to make sure it'll take input from the front like you want?

**Comment by Sly0511** [comment]
I actually solved this problem another way.
While it still persists as originally attempted, I've gone ahead and used
logistical sorters, however it didnt pull potions either, not until I
ticked the "single" option.

But before doing so, I went on creative and attempted to pu from 1 chest,
over to another, no RS involved, I put 1 glowstone and 3 healing potions,
connecting the 2 chests I placed a logistical pipe, set it to pull on the
chest with contents, push on the chest without them, and it had only moved
the glowstone, once again failing to move.

Then I tested my current setup which ill drop a screenshot later, using
logistical sorters, this works with some filters and that single option
ticked on, nothing else required.

**Comment by pupnewfster** [comment]
Which potions in particular are you trying to insert? I am testing this in 1.18 (as we are no longer actively developing for 1.16) and I am unable to reproduce this.

**Comment by Sly0511** [comment]
Healing I potions...but ye I am sorry I bothered you with an old version I
managed to get it working using logistical sorters

**Comment by pupnewfster** [comment]
Yeah I am not able to reproduce this in 1.18 so I am going to go ahead and close this as if I had to guess even in 1.16 it might only happen as some sort of mod interaction. presuming you are on the latest 1.16 version. (10.1.2)
